google_drive/google_docs.py

```diff
@@ -54,43 +53,87 @@ class GoogleDocumentBlock(XBlock):
             </iframe>
         """))
 
-    def student_view(self, context={}):
+    alt_text = String(
+        display_name="Alternative Text",
+        help=(
+            "In situations where image is not available to the reader, the alternative "
+            "text ensures that no information or functionality is lost."
+        ),
+        scope=Scope.settings,
+        default=""
+    )
+
+    # Context argument is specified for xblocks, but we are not using herein
+    def student_view(self, context):  # pylint: disable=unused-argument
         """
         Player view, displayed to the student
         """
 
         fragment = Fragment()
-        context.update({
-            "self": self,
-        })
-        fragment.add_content(render_template('/templates/html/google_docs.html', context))
-        fragment.add_css(load_resource('public/css/google_docs.css'))
-        fragment.add_javascript(load_resource('public/js/google_docs.js'))
+
+        fragment.add_content(RESOURCE_LOADER.render_template('/templates/html/google_docs.html', {"self": self}))
+        fragment.add_css(RESOURCE_LOADER.load_unicode('public/css/google_docs.css'))
+        fragment.add_javascript(RESOURCE_LOADER.load_unicode('public/js/google_docs.js'))
 
         fragment.initialize_js('GoogleDocumentBlock')
 
         return fragment
 
-    def studio_view(self, context):
+    # Context argument is specified for xblocks, but we are not using herein
+    def studio_view(self, context):  # pylint: disable=unused-argument
         """
         Editing view in Studio
         """
         fragment = Fragment()
-        fragment.add_content(render_template('/templates/html/google_docs_edit.html', {
+        # Need to access protected members of fields to get their default value
+        fragment.add_content(RESOURCE_LOADER.render_template('/templates/html/google_docs_edit.html', {
```

Comment:
I noticed that a handful of these template references are duplicated in the calendar module -- does it make sense to move them into configuration or some sort of object/property instead of having the raw strings/refs throughout the code?